RGPosadas/LitterRally
# TASK 59: Save Channels

This PR closes #210  

To test this PR:
- Run `nx serve mull-api`
- Comment the UseGuards in the post resolver and channel resolver
- Create a few channels (Make sure event with id exist): 
```graphql
mutation {
  createChannel(input: {
    name:"private",
    rights: 0,
  }, eventId: 1)
}
```
- Setup a few subscribers linked with different channels (atleast 2 with different channel ids): 
```graphql
subscription {
  postAdded(channelId: 3) {
    message
  }
}
``` 
- Create a couple of posts linked to the channels that you subscribed with:
```graphql
mutation createPost($post: CreatePostInput!) {
  post(post: $post){
    message
  }
}
```
Example Query Variables for creating posts:
```JSON
{
  "post": {
    "createdTime": "2021-02-28T16:11:00.000Z",
    "message": "hello there!",
    "channel": {
      "id": 1
    },
    "user": {
      "id": 1
    }
  }
}

```
